test/features/deleting_an_article_test.rb

```diff
@@ -0,0 +1,11 @@
+require "test_helper"
+
+feature "Deleting an article" do
+  scenario "is successful when user clicks destroy link" do
+    article = articles(:two)
+    visit article_path(article)
+    find('body').must_have_content "Second Post"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.